base/logger/DESCRIPTION

```diff
@@ -28,5 +28,6 @@ Imports:
 Suggests: testthat
 License: BSD_3_clause + file LICENSE
 Encoding: UTF-8
+LazyData: true
```

Comment:
This line should be reverted: It's true there used to be a LazyData line in every DESCRIPTION file, but recent versions of R have started recommending that it be removed when the package doesn't have any data to lazy-load.

Reply:
Got it.